index.js

```diff
@@ -8,9 +8,13 @@ const pluginDefaults = {
     'default-src': ['https:', 'unsafe-inline', 'unsafe-eval'],
     'report-uri': '/csp_reports'
   },
-  headerKey: 'Content-Security-Policy-Report-Only'
+  headerKey: 'Content-Security-Policy-Report-Only',
```

Comment:
I feel like it makes more sense if we have 'reportErrors' and 'upgradeInsecureRequests', and they can just set either or both of them to false.